src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -273,10 +273,18 @@ private void InstrumentType(TypeDefinition type)
             foreach (var method in methods)
             {
                 MethodDefinition actualMethod = method;
+                IEnumerable<CustomAttribute> customAttributes = method.CustomAttributes;
                 if (InstrumentationHelper.IsLocalMethod(method.Name))
                     actualMethod = methods.FirstOrDefault(m => m.Name == method.Name.Split('>')[0].Substring(1)) ?? method;
 
-                if (!actualMethod.CustomAttributes.Any(IsExcludeAttribute))
+                if (actualMethod.IsGetter || actualMethod.IsSetter)
+                {
+                    var prop = type.Properties.FirstOrDefault(p => p.GetMethod.FullName.Equals(actualMethod.FullName));
```

Comment:
nit: please use full name `PropertyDefinition` and not `var` when assignment expression does not decribe in a clear way a type, it's unreadable.

Reply:
sure it's an easy one to change, fixed.